### ISSUE TITLE
Unable to use cropper when accessibility mode is on

### ISSUE THREAD

**Issue by spaceface2121** [hnvn/flutter_image_cropper#288]
It is impossible to drag the crop rectangle with voiceover or talkback enabled.
